cloudflare/resource_cloudflare_ruleset.go

```diff
@@ -45,35 +62,41 @@ func resourceCloudflareRulesetCreate(d *schema.ResourceData, meta interface{}) e
 		Name:        rulesetName,
 		Description: rulesetDescription,
 		Kind:        rulesetKind,
-		Phase:       d.Get("phase").(string),
+		Phase:       rulesetPhase,
 	}
 
 	rules, err := buildRulesetRulesFromResource(d)
 	if err != nil {
-		return errors.Wrap(err, fmt.Sprintf("error building ruleset from resource"))
+		return errors.Errorf("error building ruleset rules from resource: %s", err)
```

Comment:
`fmt.Errorf` and `%w`?

Same comment applies for other modifications in the file.

https://go.dev/blog/go1.13-errors

Reply:
I originally took this approach and I ended up with a circular `%w directive can only be used in Errorf` style error despite using Errorf! I’ll have another dig into this tomorrow after ☕ and see if I can make sense of it. Otherwise, we’ll roll with this as it works.